medical-system/src/components/shared/Table.tsx

```diff
@@ -41,21 +41,9 @@ export const fetchPatients = async () => {
   });
 };
```

Comment:
To do wyrzucenia do jakeigoś osobnego folderu `services` lub coś tego typu. Może się ta funkcja przyda też w innym pliku i będzie problem.